docs/openeuropa-components.md

```diff
@@ -8,280 +8,80 @@ All code is distributed on [Packagist][3] and released under the [EUPL-1.2 licen
-            <a title="Build" href="https://drone.fpfis.eu/openeuropa/oe_search"><img src="https://drone.fpfis.eu/api/badges/openeuropa/oe_search/status.svg"/></a>
-            <a title="Downloads" href="https://packagist.org/packages/openeuropa/oe_search"><img src="https://img.shields.io/packagist/dt/openeuropa/oe_search.svg"/></a>
-        </td>
- <td><a href="https://github.com/sergepavle">Sergii Pavlenko</a></td>
-    </tr>
-    <tr>
-        <td>
-            <a title="Repository" href="https://github.com/openeuropa/oe_theme"><b>OpenEuropa Theme</b></a><br/>
-            Drupal 8 theme based on the Europa Component Library.
-        </td>
-        <td>
-            <a title="Version" href="https://packagist.org/packages/openeuropa/oe_theme"><img src="https://img.shields.io/packagist/v/openeuropa/oe_theme.svg"/></a>
-            <a title="Build" href="https://drone.fpfis.eu/openeuropa/oe_theme"><img src="https://drone.fpfis.eu/api/badges/openeuropa/oe_theme/status.svg"/></a>
-            <a title="Downloads" href="https://packagist.org/packages/openeuropa/oe_theme"><img src="https://img.shields.io/packagist/dt/openeuropa/oe_theme.svg"/></a>
-        </td>
- <td><a href="https://github.com/ademarco">Antonio De Marco</a><br/>
-            <a href="https://github.com/sergepavle">Sergii Pavlenko</a></td>
-    </tr>
-    <tr>
-        <td>
-            <a title="Repository" href="https://github.com/openeuropa/oe_webtools"><b>OpenEuropa Webtools</b></a><br/>
-            Drupal module providing webtools analytics integration such as Matomo (formerly PIWIK).
-        </td>
-        <td width="250">
-            <a title="Version" href="https://packagist.org/packages/openeuropa/oe_webtools"><img src="https://img.shields.io/packagist/v/openeuropa/oe_webtools.svg"/></a>
-            <a title="Build" href="https://drone.fpfis.eu/openeuropa/oe_webtools"><img src="https://drone.fpfis.eu/api/badges/openeuropa/oe_webtools/status.svg"/></a>
-            <a title="Downloads" href="https://packagist.org/packages/openeuropa/oe_webtools"><img src="https://img.shields.io/packagist/dt/openeuropa/oe_webtools.svg"/></a>
-        </td>
- <td><a href="https://github.com/imanoleguskiza">Imanol Eguskiza</a><br/>
-            <a href="https://github.com/upchuk">Daniel Sipos</a></td>
-    </tr>
-</table>
+Components are divided by track, each track has a few OpenEuropa team members responsible for it.
+
+### Editorial track
+
+Maintainers: [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Diogo](https://github.com/dxvargas)
+  
+Components:
+- [OpenEuropa Editorial](https://github.com/openeuropa/oe_editorial)
+
+### Multilingual track
+
+Maintainers: [Daniel](https://github.com/upchuk), [Diogo](https://github.com/dxvargas), [Imanol](https://github.com/imanoleguskiza)
+
+Components:
+- [OpenEuropa Multilingual](https://github.com/openeuropa/oe_multilingual)
+- [OpenEuropa Translation](https://github.com/openeuropa/oe_translation)
+
```

Comment:
epoetry and poetry client is missing.

Reply:
oe_epoetry is missing too

Reply:
Those two are not supposed to be used, that's why we are not listing them.